components/ProblemEditor/MetadataForm/MetadataForm.tsx

```diff
@@ -125,6 +136,49 @@ export const MetadataForm = (props: Props) => {
                 <Field component={IntegratedDatePicker} name="dueDate" />
               </LocalizationProvider>
             </Box>
+            <TextField
+              name="author"
+              label="Author"
+              value={values.author}
+              onChange={handleChange}
+              onBlur={handleBlur}
+              error={touched.author && Boolean(errors.author)}
+              helperText={touched.author && errors.author}
+              sx={{ width: "50%" }}
+            />
+            {/* <TextField
+              name="difficulty"
+              label="Difficulty"
+              value={values.difficulty}
+              onChange={handleChange}
+              onBlur={handleBlur}
+              error={touched.author && Boolean(errors.difficulty)}
+              helperText={touched.difficulty && errors.difficulty}
+              sx={{ width: "50%" }}
+            /> */}
+
+            <FormControl fullWidth sx={{ width: "25%" }}>
+              <InputLabel id="difficulty-label">Difficulty</InputLabel>
+              <Select
+                labelId="difficulty-label"
+                id="difficulty-select"
+                value={values.difficulty}
+                label="Difficulty"
+                onChange={(e) =>
+                  setFieldValue("difficulty", e.target.value, false)
+                }
+              >
+                <MenuItem value="Easy (15-29 minutes)">
```

Comment:
I think we should be storing these in the backend as "Easy", "Medium",... instead of with the times "Easy (15-29 minutes)" in the backend. Depending on the difficulty word we can change the display on the frontend or have an enum in the backend to determine that

otherwise if we ever want to change minutes for each difficulty we have to go through and purge prev values